tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -222,19 +240,26 @@ <h3>No profile data was found.</h3>
       if (oldDatasets != null || this.selected == null) {
         // Select the first dataset by default.
         this.set('selectedDataset', 0);
+        this.set('currentSelected', 0);
       }
     },
     _selectedDatasetChanged: function(newDataset, oldDataset) {
       if (this._datasets) {
         // Display the first tool by default when switching to another run.
-        this.set('selectedTool', 0);
+        this.set('currentSelected', (newDataset << 16));
```

Comment:
Same here. Could you comment on what the bit shift is doing?

Reply:
explained in comments of  currentSelected.